### PR TITLE
Chat Icons

### PR DESCRIPTION
**Pull request by TheConnMan**
image

When adding the heart icons to chat I thought it might be cool to add additional icons to distinguish types of machine chat.

### CODE REVIEW

**Comment by TheConnMan** [comment]
@Createor Could use some feedback from you as well.

File: api/services/ChatService.js [hunk 1]
```diff
@@ -17,20 +17,20 @@ function addUserMessage(chat) {
   addMessage(chat);
 }
 
-function addMachineMessage(message, username) {
+function addMachineMessage(message, username, type='machine') {
```
Comment (SirZach) [inline api/services/ChatService.js]: :)

File: assets/components/chat.html [hunk 2]
```diff
@@ -20,10 +20,20 @@ <h2 class="ui dividing header">Chat
           <div am-time-ago="{{$ctrl.getTime(chat)}}" class="date"></div>
         </div>
         <div class="extra text chat {{chat.type === 'user' ? '' : 'content minimal'}}">
+          <i class="plus icon" ng-if="chat.type === 'addVideo'"></i>
```
Comment (SirZach) [inline assets/components/chat.html]: this is what i don't like about angular. i guess in the future this could be cleaned up to a <chat-icon> component

Comment (TheConnMan) [inline assets/components/chat.html]: That's what I was planning on doing, pulling this and the `{{chat.message}}` into a new component. I made a heart component in the other PR.

**Comment by Createor** [comment]
This is a great improvement. Small change, graphically, but very pleasing to the eye. 
Can the icons be modified at all, or is this wysiwyg?

**Comment by TheConnMan** [comment]
Feel free to recommend other icons: http://semantic-ui.com/elements/icon.html

**Comment by Createor** [comment]
What about just `"user icon"` for user entered and `"eject icon"` for user left?

**Comment by TheConnMan** [comment]
I'd be fine with those. @SirZach, any opinion on those icons?

**Pull request merged**